airflow/contrib/hooks/sagemaker_hook.py

```diff
@@ -232,15 +234,20 @@ def get_conn(self):
 
     def get_log_conn(self):
         """
-        Establish an AWS connection for retrieving logs during training
+        This method is deprecated. Please use `airflow.contrib.hooks.AwsLogsHook.get_conn` instead.
 
         :rtype: CloudWatchLogs.Client
         """
-        config = botocore.config.Config(retries={'max_attempts': 15})
-        return self.get_client_type('logs', config=config)
+        warnings.warn("Method `get_log_conn` has been deprecated. "
+                      "Please use `airflow.contrib.hooks.AwsLogsHook.get_conn` instead.",
+                      category=DeprecationWarning)
```

Comment:
```suggestion
                      category=DeprecationWarning,
                      stacklevel=2)
```

That way the location will be shown from our caller, not this location.